src/components/LeetcodeText.js

```diff
@@ -0,0 +1,27 @@
+import styles from '../css/LeetcodeText.module.css'
```

Comment:
Add a media query so you can use the useMedia hook.

Import: `import useMediaQuery from "../utils/useMediaQuery`
Defining it: const `isDesktop = useMediaQuery('min-width: 600px')
Check out the other components (Note or UpcomingEvent)